jenkins-scripts/lib/check_graphic_card.bash

```diff
@@ -80,6 +83,8 @@ if $GPU_SUPPORT_NEEDED; then
       echo "GPU support needed by the script but DISPLAY var is empty"
       # Try to restart lightdm. It should stop the script in the case of failure
       sudo service lightdm restart
```

Comment:
Not sure if the systemd wrapper around `service` is still in place. We probably want to change it to use `systemctl`.

Reply:
I believe that it is, tested empirically, and I left this as is in order to retain support for past systems. But realistically our infra won't run on non-systemd-based hosts so there is no reason not to make the change which is in a2ff47eae866187254f91936e9a70b8005377029